src/odemis/acq/align/autofocus.py

```diff
@@ -979,7 +979,7 @@ def updateProgress(subf, start, end):
         future.set_progress(end=time.time() + sum(future._actions_time))
 
         logging.debug("Turning off the light")
-        bl.power.value = bl.power.range[0]
+        bl.power.value = list(bl.power.range[0])
```

Comment:
It's not needed to convert to list, right? If so, please revert this line and the l.997.